services/app-api/handlers/reports/create.ts

```diff
@@ -196,6 +200,39 @@ export const createReport = handler(
      * to create a different SAR and attach all of its fieldData to the SAR Submissions FieldData
      */
 
+    // Validate the metadata for the submission
+    const validatedMetadata = await validateData(metadataValidationSchema, {
+      ...unvalidatedMetadata,
+    });
+
+    // Return INVALID_DATA error if metadata is not valid.
+    if (!validatedMetadata) {
```

Comment:
Tried to find a scenario where this would be true, but it seems that passing in undeclared fields still passes validation. Then passing in wrongly typed values causes an error to be thrown, so this should be in a try-catch block instead.

Reply:
Good catch! Although it looks like I added this it was actually elsewhere in the file and just got moved around. I made a ticket to follow up on this [CMDCT-3988](https://jiraent.cms.gov/browse/CMDCT-3988)